web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -121,7 +121,34 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
 
         this._loadMenuItemsFromHandlers(processedValues, collection);
 
-        return this._applyTemplate(tpl, processedValues);
+        if (this._spatialSubsetIntersect(collection)) {
+            return this._applyTemplate(tpl, processedValues);
+        } else {
+            return this._emptyMsg();
+        }
+    },
+
+    _spatialSubsetIntersect: function(collection) {
```

Comment:
For me there are a couple of issues.
  - should the check be performed for all data sets and protocols - not only ncwms? If not then the naming should be changed to reflect that.
  - should the extent calculation be being done in a view class? For example if the predicate behavior was factored out then it would be available to show a pop-up at stage 2 for instance. Just an example, not saying we should do this. In any case I feel this is more model code, that view code.